src/main/scala/tech/cryptonomic/conseil/Lorre.scala

```diff
@@ -14,12 +15,21 @@ import scala.util.{Failure, Success, Try}
   */
 object Lorre extends App with LazyLogging {
 
+  private val conf = ConfigFactory.load
+  val refreshIntervalInSeconds = conf.getInt("lorre.sleepIntervalInSeconds")
```

Comment:
The variable name should ideally match the renamed configuration parameter.

Reply:
Fixed.